src/test/ui/nll/closure-requirements/region-ebr-does-not-outlive-static.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
```

Comment:
can't you find some compile-fail tests and run *them* with NLL?

Reply:
Conceivably. =) Yeah, I'll search.